app/views/admin/loans/questionnaires/_questionnaire_group.html.slim

```diff
@@ -2,13 +2,13 @@
   / should probably be refactored someday to happen in the model
   div
     - optional_encountered = false
-    - optionals_exist = roots.any? { |i| !i.required_for?(@loan) }
-    - requireds_exist = roots.any? { |i| i.required_for?(@loan) }
+    - optionals_exist = roots.any? { |i| !i.required? }
+    - requireds_exist = roots.any? { |i| i.required? }
```

Comment:
Can use abbreviated syntax for this and also for the above if you add an `optional?` method that just negates `required?`